colobot-base/src/ui/screen/screen_io_read.cpp

```diff
@@ -168,10 +171,22 @@ bool CScreenIORead::EventProcess(const Event &event)
 
     if ( event.type == EVENT_INTERFACE_IOREAD )
     {
-        if(IOReadScene() && m_inSimulation)
+        auto doRead = [&]() {
+            if(IOReadScene() && m_inSimulation)
+            {
+                m_main->StopSuspend();
+                m_main->ChangePhase(PHASE_SIMUL);
+            }
+        };
+        std::optional<std::filesystem::path> name = GetSceneName();
+        if (!name.has_value()) return false;
+        if (IsVersionSaveSupported(name.value()))
         {
-            m_main->StopSuspend();
-            m_main->ChangePhase(PHASE_SIMUL);
+            doRead();
+        }
+        else
+        {
+            m_dialog->StartQuestion("Version unsupported. Open anyway?", false, false, false, doRead);
```

Comment:
This text should be translatable (see [restext.cpp](https://github.com/colobot/colobot/blob/dev/colobot-base/src/common/restext.cpp), `update-pot`).